bin/neurondemo.in

```diff
@@ -24,9 +24,9 @@ fi
 
 export NRNDEMO
 
-# TODO : see https://github.com/neuronsimulator/nrn/issues/494
-if [ ! -L "${NRNDEMO}neuron" ] ; then
+if [ ! -f "${NRNDEMO}neuron" ] ; then
 	(cd "${NRNDEMO}release" ; "${NRNBIN}nrnivmodl" ; cd .. ; rm -f neuron ; ln -s release/${MODSUBDIR}/special neuron)
 fi
```

Comment:
@nrnhines : this `-L` was done for the reason mentioned in #494. IIRC, you have worked on neurodemo to be shipped with wheel but we should download wheels from CI of this PR and should make sure they work as expected.

Reply:
Ok, I see OSX CI is failing. This is what I was referring to in the above comment.

Reply:
My guess is that the full cherry-pick of the #468 merge is needed but I'm starting out confused and not at all certain of that.  The Symptom of the neurondemo failure from the pkg is that neurondemo tries to rebuild the libnrnmech.so even though it already exists.

Reply:
@nrnhines : do you mean #765 and not #468?

Reply:
Correct, #765 . I'll likely close this and try again with that cherry-pick.